requirements.txt

```diff
@@ -1,17 +1,96 @@
-flask
-flask-cors
-flask_httpauth
-GPy
-keras
-matplotlib
-networkx
-neo4j
-numpy
-pandas
-plotly
-py2neo
+absl-py==0.11.0
+astroid==2.4.2
+astunparse==1.6.3
+blis==0.4.1
+cachetools==4.1.1
+catalogue==1.0.0
+certifi==2020.11.8
+cffi==1.14.3
+chardet==3.0.4
+click==7.1.2
+cryptography==3.2.1
+cycler==0.10.0
+cymem==2.0.4
+decorator==4.4.2
+docker==4.3.1
+en-core-web-md @ https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.3.1/en_core_web_md-2.3.1.tar.gz
+en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.1/en_core_web_sm-2.3.1.tar.gz
+english==2020.7.0
+Flask==1.1.2
+Flask-Cors==3.0.9
+Flask-HTTPAuth==4.1.0
+gast==0.3.3
+google-auth==1.23.0
+google-auth-oauthlib==0.4.2
+google-pasta==0.2.0
+GPy==1.9.9
+grpcio==1.33.2
+h5py==2.10.0
+idna==2.10
+isort==5.6.4
+itsdangerous==1.1.0
+Jinja2==2.11.2
+joblib==0.17.0
+Keras==2.4.3
+Keras-Preprocessing==1.1.2
+kiwisolver==1.3.1
+lazy-object-proxy==1.4.3
+Markdown==3.3.3
+MarkupSafe==1.1.1
+matplotlib==3.3.2
+mccabe==0.6.1
+monotonic==1.5
+murmurhash==1.0.4
+neo4j==4.1.1
+neotime==1.7.4
```

Comment:
Do you use neo4j or is this from an earlier version of franky?

Reply:
An earlier version. I didn't remove any of the now redundant packages. This is just a pip freeze of the old requirements.txt with the couple of extra packages I did add.